### PR TITLE
[Specimen] GetcenterId faster

### PR DESCRIPTION
**Pull request by ridz1208**

### CODE REVIEW

**Comment by driusan** [comment]
can you send this to aces/Loris#8085? I'm afraid of losing the change if that gets merged..

**Comment by ridz1208** [comment]
@driusan I will as soon as I get confirmation that its working properly

File: php/specimendao.class.inc [hunk 1]
```diff
@@ -379,6 +381,7 @@ class SpecimenDAO extends \LORIS\Data\ProvisionerInstance
             foreach ($row as $value) {
                 $pA[$row['protocolId']][$row['attributeId']]['label']           = $row['label'];
                 $pA[$row['protocolId']][$row['attributeId']]['datatypeId']      = $row['datatypeId'];
+                $pA[$row['protocolId']][$row['attributeId']]['refTableId']      = $row['refTableId'];
```
Comment (HenriRabalais) [inline php/specimendao.class.inc:384]: @ridz1208 do you recall the motivation for this change? It doesn't seem related to anything else in the pR
